.github/workflows/linux.yml

```diff
@@ -2,11 +2,11 @@ name: Check Linux Packages
 on:
   push:
```

Comment:
while you're here would you mind removing the whole `push` block?

It seems necessary to run the build/test again especially as nobody is monitoring it's status once it's been merged

Reply:
done